doc/src/records/top-members.md

```diff
@@ -532,6 +532,7 @@ DEFAULT,        Default: tSky from weather file if available else Berdahl-Martin
 BERDAHLMARTIN,  Berdahl-Martin (tSky depends on dew point&comma; cloud cover&comma; and hour)
 DRYBULB,   	   tSky = dry-bulb temperature (for testing)
 BLAST,          Blast model (tSky depends on dry-bulb)
+IRHORIZ,        Experimental model that derives tSky for horizonal infrared data from the weather file (available on some EPW files only)
```

Comment:
It's not experimental once it's merged.

Reply:
1. Reworded.
2. I don't agree that there can be no experimental models in merged code.  New stuff has to be merged to allow external users to test.